classic-commerce.php

```diff
@@ -30,33 +27,24 @@ function cc_wc_already_active_notice() {
 	echo esc_html__( 'You must deactivate WooCommerce before activating Classic Commerce.', 'classic-commerce' );
 	echo '</p></div>';
 }
-
-if ( file_exists( WP_PLUGIN_DIR . '/woocommerce/woocommerce.php' ) && file_exists( WP_PLUGIN_DIR . '/woocommerce/includes/class-woocommerce.php' ) && file_exists( WP_PLUGIN_DIR . '/woocommerce/includes/admin/class-wc-admin.php' ) ) {
-
+if ( file_exists( WP_PLUGIN_DIR . '/woocommerce/woocommerce.php' ) && file_exists( WP_PLUGIN_DIR . '/woocommerce/includes/class-woocommerce.php' ) && is_plugin_active( 'woocommerce/woocommerce.php' ) ) {
```

Comment:
I think it's better to use `function_exists( 'wc' )` here as per the code I originally suggested in https://github.com/ClassicPress-research/classic-commerce/issues/152#issuecomment-568948689

One reason is that some people rename the WooCommerce plugin folder, another reason is that it's simpler and it would have avoided several of the issues CC has been experiencing related to this code.

Reply:
One of the issues with using `function_exists( 'wc' )` is that this will return `true` if either WC or CC is active. We need to return `true` only if WC is active. 

That's the reason for using `is_plugin_active( 'woocommerce/woocommerce.php' )` along with checks for WC-specific files instead of using `function_exists( 'wc' )`.

I can't see a perfect solution for this. There are always going to be some edge cases. People renaming the `woocommerce` folder being one of them.

Reply:
It shouldn't return true if CC is active, because at this point in the load process, the `wc` function hasn't been defined yet.

Reply:
I just did a quick test, and in order for this to work, `function wc` would need to be defined inside an `if` statement. I still think this is probably the best (simplest) solution.

Reply:
Ok, I see the problem now. Classic Commerce loads before WooCommerce on my test site, so the `wc` function does not exist at the time this check is performed.

So the approach of checking for WC plugin files + activation is the correct one. Sorry for the confusion!

Reply:
No probs at all and thanks for checking it. It's not a very satisfactory solution but I can't think of a better one.